packages/web-client/README.md

```diff
@@ -1,3 +1,69 @@
 # web-client
 
-This is experimental.
+The `web-client` is a standalone package that allows you to interact with the [ownCloud Infinite Scale (oCIS)](https://github.com/owncloud/ocis/) APIs via TypeScript. It also provides convenient utilities and interfaces that are useful when working with these APIs.
```

Comment:
I'd like to add to this, that `web-client` also serves as an abstraction layer for the APIs. We never let the user deal with raw requests and responses. Instead we're transforming data into objects with useful types and helpers (like e.g. permission check helpers).

Reply:
This is mostly the case for the webdav part of the client. There you could state as an example that consumers of the client would never need to deal with the webdav domain knowledge of what a `PROPFIND` is, instead we offer functions like `listFiles` and `getFileInfo`.

Reply:
> I'd like to add to this, that web-client also serves as an abstraction layer for the APIs. We never let the user deal with raw requests and responses. Instead we're transforming data into objects with useful types and helpers (like e.g. permission check helpers).

For Graph this is not true though :/ The Graph abstraction just wraps the generated SDK but adds nothing to it. (something I'd really like to change in the future).

A general note might be misleading because of that. Maybe a dedicated paragraph just for WebDAV underneath this one?

Reply:
Or write a general note and with an exception for Graph 🤔

Reply:
...or just a general note without any reference to one of the layers 🙈 I mean (or I hope) we will adjust the Graph layer at some point in time in the future to reflect this.

Reply:
I'd prefer the general note and an exception for Graph, because I like to imagine the web-client as an abstraction layer.